include/xvega-bindings/xvega_bindings.hpp

```diff
@@ -189,6 +197,40 @@ namespace xv_bindings
 
             return it;
         }
+
+        /* Implementation of a visitor for xv::xany which is a xtl::any */
+        using visitor_map_type = std::unordered_map<std::type_index, std::function<void(xtl::any const&)>>;
+
+        template<typename U, typename F>
+        inline std::pair<const std::type_index, std::function<void(xtl::any const&)> >
+            to_any_visitor(F const &f)
+        {
+            return {
+                std::type_index(typeid(U)),
+                [g = f](xtl::any const &a)
```

Comment:
I think you can simply capture f by value and directly use it after. Besides, initializing a local variable in a lambda is C++17

Reply:
Thanks! :)